src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -92,6 +92,18 @@ protected virtual void Initialize()
                 File.WriteAllText(hostConfigFilePath, "{}");
             }
 
+            if (ScriptConfig.HostConfig.IsDevelopment)
+            {
+                ScriptConfig.HostConfig.UseDevelopmentSettings();
+            }
+            else
+            {
+                // TEMP: Until https://github.com/Azure/azure-webjobs-sdk-script/issues/100 is addressed
+                // we're using some presets that are a good middle ground
+                ScriptConfig.HostConfig.Queues.MaxPollingInterval = TimeSpan.FromSeconds(10);
+                ScriptConfig.HostConfig.Singleton.ListenerLockPeriod = TimeSpan.FromSeconds(15);
```

Comment:
1 minute for the Queue, and 1 minute for the listener lock. The UseDevelopmentSettings will set the Queue value to 2 seconds which is way to aggressive for anything other than short term use (i.e. debugging).